kerastuner/tuners/hyperband.py

```diff
@@ -172,7 +172,7 @@ def _populate_space(self, trial_id):
 
         # Max sweeps has been reached, no more brackets should be created.
         if (self._current_bracket == 0 and
```

Comment:
Would it be better to change this check to "<=" instead of setting it to 0 when it its negative?

Reply:
I don't think it would work, the problem is not that it can be negative. 
In 180 it would call to increment it, but the increment (195-199) will not let it go to negative, it would restart before that. A negative _current_bracket number can never reach line 174 (bc 197).
My solution would stop the restarting.

If you want it that way you suggested:
```suggestion
        if (self._current_bracket <= 0 and
```
you also need in line 197
```
        if self._current_bracket < 0 and self._current_iteration <= self.hyperband_iterations:
```

You need to stop the restarting either way.